site/_includes/layouts/home.njk

```diff
@@ -19,12 +19,12 @@
         ) }}
     <div class="gap-top-500 grid-gap-500 display-flex direction-column masonry:direction-row align-center masonry:align-start">
       <div class="display-flex direction-column grid-gap-500">
-        {% set post = helpers.findByUrl(collections.all, '/blog/insider-april-2022', 'en') %}
+        {% set post = helpers.findByUrl(collections.all, '/blog/insider-july-2022', 'en') %}
         {% if post %}
         {{ featuredCard(post, {
           icon: 'star',
           actionText: 'Discover',
-          actionUrl: '/blog/insider-april-2022/'
+          actionUrl: '/blog/insider-july-2022/'
```

Comment:
This will cause both cards on the homepage to show the same post, it will look like a duplicate. I would wait with updating "featured" card until we get another (newer) blog post published.